### PR TITLE
Service Inventory refactor to have 1 collection per subject level

### LINKED ISSUE #1100: Refactor service inventory API to use less MongoDB collections
There's a limit of 500 collections across all DBs on the plan used to host our dev links, service inventory currently uses 28 (of the total 45) collections per DB. Service inventory population is currently disabled, but unless that number is reduced then we'll be capped at only ~10 concurrent active dev links once it is enabled.

Upgrading the MongoDB Atlas tier isn't an option, would need to go up two tiers and drastically increase the cost to get past the (arbitrary?) 500 collection limit, and we don't need any of what we'd _actually_ be paying for (disk space, speed) from that.

Only alternative I could think of is to do something hacky like open a bunch of free tier projects and round-robin the dev link DBs between them.

### PR DESCRIPTION
**Pull request by ktw1016**
closes #1100

### CODE REVIEW

File: server/src/models/services/models.js [hunk 1]
```diff
@@ -148,120 +148,68 @@ export default function (model_singleton) {
...
+    top_services_application_vol_summary: [
+      TopServicesApplicationVolSummarySchema,
+    ],
+    top_services_website_visits_summary: [
+      TopServicesWebsiteVisitsSummarySchema,
+    ],
+    service_fees_summary: [ServiceFeesSummarySchema],
+    service_standards_summary: [ServiceStandardsSummarySchema],
+  });
+  const ProgramServiceSummarySchema = mongoose.Schema({
```
Comment (ktw1016) [inline server/src/models/services/models.js:179]: Could simply combine org and program as they're equivalent but just leaving it separately as dept and program may diverge in the future

Comment (Stephen-ONeil) [reply server/src/models/services/models.js:179]: Seems good to me. Maybe just to make it clearer at a glance what's common between these three schemas, as well as between the org and program schemas in particular, ou could define the common fragments above and just spread them in here?

**Review by Stephen-ONeil — CHANGES_REQUESTED** [review]
Two small comments, but feel free to push back on either if you don't agree.

**Review by Stephen-ONeil — APPROVED** [review]
:+1:

**Pull request merged**